tools/multibuild/Dockerfile.package

```diff
@@ -0,0 +1,59 @@
+# Dockerfile.package
+# A dockerfile for packaging SSH No Ports releases using docker buildx
+
+FROM atsigncompany/buildimage:3.5.2_3.6.0-149.3.beta@sha256:df67b9e3271381fc0c5b20e7350cf4de8dad6ac62e075b49b1a866c49af47409 AS build
+# Using atsigncompany/buildimage until official dart image has RISC-V support
+WORKDIR /noports
+
+# install node for later (keep at the top file to increase cache hits)
+# hadolint ignore=DL3008
+RUN apt-get update; \
+  apt-get install -y --no-install-recommends npm
+
+COPY . .
+
+# Build packages/dart/sshnoports
+WORKDIR /noports/packages/dart/sshnoports
+RUN set -eux; \
+  mkdir -p /sshnp/debug; \
+  mkdir -p /sshnp/web/admin; \
+  mkdir /tarball; \
+  dart pub get --enforce-lockfile; \
+  dart run build_runner build --delete-conflicting-outputs; \
+  dart compile exe bin/activate_cli.dart -v -o /sshnp/at_activate; \
+  dart compile exe bin/sshnp.dart -v -o /sshnp/sshnp; \
+  dart compile exe bin/npt.dart -v -o /sshnp/npt; \
+  dart compile exe bin/npp_file.dart -v -o /sshnp/npp_file; \
+  dart compile exe bin/npp_atserver.dart -v -o /sshnp/npp_atserver; \
+  dart compile exe bin/sshnpd.dart -v -o /sshnp/sshnpd; \
+  dart compile exe bin/srv.dart -v -o /sshnp/srv; \
+  dart compile exe bin/npp.dart -v -o /sshnp/npp; \
```

Comment:
This line needs to be removed, since it is now `npp_atserver` which is already there 3 lines above

Reply:
Done

Reply:
Running another multibuild [here](https://github.com/atsign-foundation/noports/actions/runs/10725814755)

Reply:
multibuild ran ok